### DISCUSSION TITLE
HALMET has no solder jumpers for pullup/pulldown?

### DISCUSSION THREAD

**Discussion by CD33-Jay** [hatlabs/discussions#113] [General]
Just received my order for SH-ESp32 and HALMET.  Look great.
The HALMET documentation and schematic show solder jumpers for pullup and pulldown resistors on the digital inputs.  The version v1.0.0-a1 board I received does not have these jumpers.   Am I missing something?
Thanks,
Jay
![PXL_20240403_212937858 resized](https://github.com/hatlabs/discussions/assets/112293498/e3679dc1-6848-4259-8532-36f592f1784a)

**Comment by mairas** [comment]
Oof, you've by accident received a proto version of the board. I'll email you about providing a replacement.

**Reply by mairas** [reply]
Except that I can't - I'm not sure who you are! :-D Can you please email me at matti.airas@hatlabs.fi and we'll sort this out.
